### ISSUE TITLE
Implement getters for segments and points

### ISSUE THREAD

**Issue by zhenyanghua** [zhenyanghua/MeasureTool-GoogleMaps-V3#45]
```js
let measureTool = new MeasureTool(..);

// user draw ...

measureTool.getPoints(); // return [{lat: 10, lng: 10}, {lat: 20, lng : 20} ...];
```

There are 3 reasons for this:
1. Using this method I don't have to get the `MeasureResult` only when user trigger events (`measure_start`, `measure_end`, `measure_change`).

2. Even though I've got the `MeasureResult`,  using `MeasureResult` I have to parse the `start_location` and `end_location` of every `segments` to get the points.  And If I simply push them (`start_location` and `end_location`) into an array, I would have to do the deduplication work (as every two adjacent segments will have at least one same point ), which is kind of annoying.

3. The new feature in v0.3.0 allow user to pass an array of points to `start()`, like
```js
`measureTool.start([{lat: 47.94796, lng: -116.70797}, {lat: 47.95028, lng: -116.69907}])`
```

_Originally posted by @profileCaren in https://github.com/zhenyanghua/MeasureTool-GoogleMaps-V3/issues/31#issuecomment-436547928_

**Comment by zhenyanghua** [comment]
will add getters to expose segments and points in next release.

**Comment by profileCaren** [comment]
@zhenyanghua Thanks :D

**Comment by zhenyanghua** [comment]
Implemented in v0.4.0
